### PR TITLE
feat: add syntax highlighting to homepage

### PR DESCRIPTION
**Pull request by nandorojo**
I couldn't preview it locally without adding a bunch of env variables, but assuming Vercel generates a preview link of this PR, we can preview it there.

Uses [`bright`](https://bright.codehike.org/) for RSC-compatible syntax-highlighting.

### CODE REVIEW

**Comment by m2rads** [comment]
@nandorojo Could you please provide a screenshot of how it looks now?

**Comment by nandorojo** [comment]
Yeah I want to except the app crashes when I try to run it since I don't have env variables set. Specifically clerk throws

**Comment by m2rads** [comment]
Can you try running the setup script in `package.json`? That should help you to add the required env vars. For clerk you would need these two: 
```
NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
CLERK_SECRET_KEY=
```

All the others are explained in `.env.example`. Let me know if you have any questions.

**Comment by m2rads** [comment]
There is a build issue in preview env: 

image

**Comment by nandorojo** [comment]
Hm yeah I wonder if I can just use fake clerk creds let me try this

**Comment by slavingia** [comment]
^ if that works maybe we provide them by default so people can easily run the website without needing to setup Clerk.

File: app/(dashboard)/page.tsx [hunk 1]
```diff
@@ -28,12 +29,12 @@ export default function HomePage() {
               Write tests in plain English
             </h3>
             <pre className="bg-gray-800 text-gray-100 p-4 rounded-lg overflow-x-auto">
-              <code>{`import { shortest } from '@antiwork/shortest'
+              <Code language="js">{`import { shortest } from '@antiwork/shortest'
```
Comment (slavingia) [inline app/(dashboard)/page.tsx:32]:
```suggestion
              <Code lang="js">{`import { shortest } from '@antiwork/shortest'
```

**Comment by slavingia** [comment]
Should be lang instead of language, ha. Committed a fix

**Comment by nandorojo** [comment]
Ty lol my bad. Got stuck on the env
